apps/src/lib/node/ledger/shell/vote_extensions/eth_events.rs

```diff
@@ -33,6 +33,7 @@ where
     ///  * There are no duplicate Ethereum events in this vote extension, and
     ///    the events are sorted in ascending order.
     #[inline]
+    #[allow(dead_code)]
```

Comment:
I want to suggest we remove dead code from our codebase, but that's a separate code style issue. e.g. we would now remove `validate_eth_events_vext` and `validate_valset_upd_vext` in this PR if they aren't needed anymore